lighthouse-core/report/html/renderer/util.js

```diff
@@ -665,8 +665,10 @@ Util.UIStrings = {
   /** Title of the lab data section of the Performance category. Within this section are various speed metrics which quantify the pageload performance into values presented in seconds and milliseconds. "Lab" is an abbreviated form of "laboratory", and refers to the fact that the data is from a controlled test of a website, not measurements from real users visiting that site.  */
   labDataTitle: 'Lab Data',
 
-  /** This label is for a checkbox above a table of items loaded by a web page. The checkbox is used to show or hide third-party (or "3rd-party") resources in the table, where "third-party resources" refers to items loaded by a web page from URLs that aren't controlled by the owner of the web page. */
-  thirdPartyResourcesLabel: 'Show 3rd-party resources',
+  /** This label is for a checkbox above a table of items loaded by a web page. This option is used to show all resources in the table. The term `resources` may be elided, as long as it is used in the "exclude" label. */
```

Comment:
yea but ... i want `resources` elided in english so can't really include it in the string here.

Reply:
> yea but ... i want `resources` elided in english so can't really include it in the string here.

Ah, are you saying this is short for "Show all resources" but "resources" may be elided? Maybe better to leave it at "This option is used to show all resources in the table" since that gives the context (the first response to "The term `resources` may be elided" is "there is no term `resources` in the string").

> as long as it is used in the "exclude" label

translators get the strings individually and don't necessarily handle other strings or are even able to see them.

Reply:
I'm just afraid that doing that will result in all languages other than English showing resources in both options, which will look worse.

Reply:
@exterkamp for feedback